command/generate.go

```diff
@@ -36,23 +35,28 @@ func (c *generateTask) ParseParams(params map[string]interface{}) error {
 }
 
 func (c *generateTask) Execute(ctx context.Context, comm client.Communicator, logger client.LoggerProducer, conf *model.TaskConfig) error {
-
+	var err error
 	if conf.Task.Execution > 0 {
 		logger.Task().Warning("Refusing to generate tasks on an execution other than the first one")
 		return nil
 	}
+  
 	if err := util.ExpandValues(c, conf.Expansions); err != nil {
 		err = errors.Wrap(err, "error expanding params")
 		logger.Task().Error(err)
 		return err
+  }
+
+	if c.Files, err = util.BuildFileList(conf.WorkDir, c.Files...); err != nil {
+		err = errors.Wrap(err, "problem building wildcard paths")
+		logger.Task().Error(err)
```

Comment:
I think this is redundant, since the error will be logged by the caller. I think you can just `return errors.Wrap(err, "problem building wildcard paths")` in 1 line instead of these 3. You can also do the same above and below.

Reply:
> You can also do
Actually, you _should_ also do